app/ingestion/jd_mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mappings": {
+    "job_description": {
+      "properties": {
+        "companyName": {
+          "type": "keyword",
+          "store": false,
+          "index": true
+        },
+        "location": {
+          "type": "text",
```

Comment:
I guess with the way it is - you cannot do aggregations on locations... if you go with this mapping for ```location```  below, you can (And I think it is important to be able to do aggregations on the complete location string). The same goes for the title and other string fields here..
```
{
  "location": {
    "type" "text",
    "fields": {
      "keyword": {
        "type": "keyword",
        "ignore_above": 256
      }
    }
  }
}
```